doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst

```diff
@@ -338,7 +338,9 @@ Wi-Fi samples
 Other samples
 -------------
 
-|no_changes_yet_note|
+* :ref:`coremark_sample` sample:
+
+  * Updated the logging mode to the minimal mode (:kconfig:option:`CONFIG_LOG_MODE_MINIMAL`) to minimize the memory footprint of this sample.
```

Comment:
to minimize the memory footprint of this sample. ->  to minimize the memory footprint of this sample and guarantee that logging wouldn't interfere with running benchmark.